docs/DeveloperGuide.md

```diff
@@ -1,3 +1,1140 @@
+**Developer Guide**
+===================
+
+**Introduction**
+================
+
+### **What is HealthVault**
+
+HealthVault is a desktop app for managing doctor, nurse and patient
+information, optimised for use through the command line interface. This
+app is for the head nurse of a hospital, if the user can type fast, it
+is better than a traditional GUI app.
+
+### **Purpose and scope**
+
+The purpose of this developer guide is to describe the architecture and
+software design decisions for our application. This guide will cover our
+program architecture, the logical view of major components and how our
+functions work.
+
+**Setting Up and Getting Started**
+----------------------------------
+
+### **Setting up**
+
+There are 2 prerequisites for this project
+
+1.  Java 11
+
+-   Launch your terminal and type "java -version" to ensure the correct
+    > version
+
+2.  Intellij IDEA
+
+### **Getting Started**
+
+1.  Fork this repo, and clone the fork into your computer.
+
+2.  Import the project as a gradle project
+
+3.  Run Duke.main and try a few commands
+
+4.  Run the tests to ensure they all pass.
+
+**Design**
+----------
+
+**Architecture**
+----------------
+
+![](media/image2.png){width="6.267716535433071in"
```

Comment:
![image](https://user-images.githubusercontent.com/11938072/113230012-bc971880-92ca-11eb-9597-648cef6b9202.png)
There is no image under the Architecture header